lib/src/git_backend.rs

```diff
@@ -1468,13 +1468,13 @@ fn write_tree_conflict(
     .collect_vec();
     let readme_id = repo
         .write_blob(
-            r#"This commit was made by jj, https://github.com/jj-vcs/jj.
+            r#"This commit was made by jj, https://jj-vcs.dev/.
 The commit contains file conflicts, and therefore looks wrong when used with plain
 Git or other tools that are unfamiliar with jj.
 
 The .jjconflict-* directories represent the different inputs to the conflict.
 For details, see
-https://jj-vcs.github.io/jj/prerelease/git-compatibility/#format-mapping-details
+https://docs.jj-vcs.dev/latest/git-compatibility/#format-mapping-details
```

Comment:
Why did "prerelease" change to "latest"? If it's intentional, please put it in a separate commit and explain why in the commit description.

Reply:
Ah, I think this one was sort of a mistake; at least, it's the only one that links to prerelease specifically. Was there a reason for that? It's in latest at this point. Happy to move it into another commit or just move back to `/prerelease` if that was what was actually desired.

Reply:
actually, sure, let's keep it the same for this one regardless, I am vaguely curious but if it needs to be changed, could be done in a follow-up :)